dts/bindings/ieee802154/atmel,rf2xx.yaml

```diff
@@ -38,3 +38,53 @@ properties:
       description: |
         Specifies the MAC address that was assigned to the network
         device
+
+    channel-page:
+      type: int
+      enum:
+        - 0
+        - 2
+        - 5
+      default: 0
```

Comment:
since you are using DT_*PROP_OR you should remove the defaults from the binding.

Reply:
Fixed